### PR TITLE
Add a version of the rancher-webhook chart for the Rancher v2.6.3-patch1 release

### PR DESCRIPTION
**Pull request by maxsokolovsky**
Related to https://github.com/rancher/rancher/issues/36177.
This makes a chart for rancher-webhook `v0.2.4-rc7`.
It's to be used only for the Rancher release `v2.6.3-patch1`.

### CODE REVIEW

**Comment by maxsokolovsky** [comment]
Looks like there is no CI - is it configured on a per-branch basis?

**Review by thedadams — CHANGES_REQUESTED** [review]
Per "offline" discussion, version `v0.2.3-patch1` is not what we should go with here.

**Comment by maxsokolovsky** [comment]
This makes a chart for rancher-webhook `v0.2.4-rc7`.

**Review by thedadams — APPROVED** [review]

**Review by cmurphy — APPROVED** [review]
lgtm but I think @aiyengar2 should rubber-stamp it

File: packages/rancher-webhook/package.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+url: https://github.com/rancher/webhook/releases/download/v0.2.4-rc7/rancher-webhook-0.2.4-rc7.tgz
+version: 1.0.3
```
Comment (cbron) [inline packages/rancher-webhook/package.yaml:2]: Can you cut a 0.2.5-rc1 with actual new code in master, then update this in charts dev-v2.6 to be 1.0.4+0.2.5-rc1, then announce somewhere so people don't accidentally use 0.2.4.

File: release.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+rancher-webhook:
+- 1.0.3+up0.2.4-rc7
```
Comment (cbron) [inline release.yaml:2]: Just a reminder to cut final tag before we go live.

**Review by cbron — APPROVED** [review]

File: packages/rancher-webhook/package.yaml (hunk 1 above)
Comment (maxsokolovsky) [reply packages/rancher-webhook/package.yaml:2]: Will do, do you want the `v0.2.5-rc1` tag to be on the latest commit in master?

Comment (maxsokolovsky) [reply packages/rancher-webhook/package.yaml:2]: If yes, then the latest commit will have two tags: `v0.2.4-rc6` and `v0.2.5-rc1`, if that's OK...

Comment (cbron) [reply packages/rancher-webhook/package.yaml:2]: Yes thats ok. We want 0.2.4-rc6 to be forgotten.

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Pull request merged**